frontend/simple/views/CreateGroup.vue

```diff
@@ -107,28 +121,31 @@ export default {
         ctrlEl.scrollIntoView()
       }
       if (success) {
-        let entry = new Events.GroupContract({
-          authorizations: [Events.CanModifyAuths.dummyAuth()],
-          groupName: this.groupName,
-          sharedValue: this.sharedValues,
-          changePercentage: this.changePercentage,
-          memberApprovalPercentage: this.memberApprovalPercentage,
-          memberRemovalPercentage: this.memberRemovalPercentage,
-          incomeProvided: this.incomeProvided,
-          contributionPrivacy: this.contributionPrivacy,
-          founderUsername: this.$store.state.loggedIn
-        })
-        let hash = entry.toHash()
-        await backend.subscribe(hash)
-        Vue.events.$once(hash, (contractId, entry) => {
-          this.$store.commit('setCurrentGroupId', hash)
-        })
-        let res = await backend.publishLogEntry(hash, entry)
-        if (!res.ok) {
-          console.error('failed to create group contract:', res.text)
-        } else {
+        try {
+          this.errorMsg = null
+          let entry = new Events.GroupContract({
+            authorizations: [Events.CanModifyAuths.dummyAuth()],
+            groupName: this.groupName,
+            sharedValues: this.sharedValues,
+            changePercentage: this.changePercentage,
+            memberApprovalPercentage: this.memberApprovalPercentage,
+            memberRemovalPercentage: this.memberRemovalPercentage,
+            incomeProvided: this.incomeProvided,
+            contributionPrivacy: this.contributionPrivacy,
+            founderUsername: this.$store.state.loggedIn
+          })
+          let hash = entry.toHash()
+          await backend.subscribe(hash)
+          Vue.events.$once(hash, (contractId, entry) => {
+            this.$store.commit('setCurrentGroupId', hash)
+          })
+          await backend.publishLogEntry(hash, entry)
```

Comment:
Well a non 200 Ok response would be an exception and will show in the console as well even if we don't console log it.